apps/api/src/controllers/applicationController.ts

```diff
@@ -257,6 +257,102 @@ export const submitRevision = async ({ applicationId }: { applicationId: number
 	}
 };
 
+export const requestApplicationRevisionsByDac = async ({
+	applicationId,
+	role,
```

Comment:
This function probably does not need the `role` input parameter. This will only get called when the system intends to take the DAC revisions action, all the role based approval should happen before this is called. In fact, we do perform this role check in the router.

Strictly speaking this can be left in, but we should standardize how we do permission checking instead of passing partial identity information as parameters into our functions. I'll approve this for merging but will likely refactor it along with the other API auth changes.